code/style.css

```diff
@@ -1,6 +1,26 @@
 body {
   margin: 0;
+  padding: 0;
   font-family: "Courier New", Courier, monospace;
+  background-image: url(./assets/hallway.jpeg);
```

Comment:
It's far more safe approach to wrap path in parentheses. Without them, some browsers might not find the path
`url("./assets/hallway.jpeg");`